desc/objectives/_geometry.py

```diff
@@ -376,7 +379,12 @@ def build(self, use_jit=True, verbose=1):
         """
         eq = self.things[0]
         if self._grid is None:
-            grid = QuadratureGrid(L=eq.L_grid, M=eq.M_grid, N=eq.N_grid, NFP=eq.NFP)
+            grid = QuadratureGrid(
```

Comment:
Do we want a quadrature grid for surface classes? I think quadrature grid with L=1 doesn't have a node at rho=1

Reply:
good catch